src/lib/holocene/toggle-switch.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+  export let checked = false;
+  export let id: string;
+</script>
+
+<label for={id} class="switch">
+  <input {id} bind:checked type="checkbox" class="hidden" />
+  <span class="slider" />
+</label>
+
+<style lang="postcss">
+  .switch {
+    @apply relative inline-block h-8 w-[52px] rounded-[8rem];
+  }
+
+  .slider {
+    @apply absolute top-0 left-0 right-0 bottom-0 cursor-pointer rounded-[8rem] border-2 border-gray-600 bg-gray-100;
+    transition: 300ms;
```

Comment:
Could we do these with Tailwind and/or Svelte's [transition](https://svelte.dev/docs#run-time-svelte-transition) library?